synapse/lib/view.py

```diff
@@ -530,14 +584,17 @@ async def _tagPropSetConfirm(self, user, snap, splice):
         perms = ('node', 'tag', 'add', *tag.split('.'))
         self.parent._confirm(user, perms)
 
-    async def mergeAllowed(self, user=None):
+    async def mergeAllowed(self, user=None, force=False):
         '''
         Check whether a user can merge a view into its parent.
         '''
         fromlayr = self.layers[0]
         if self.parent is None:
             raise s_exc.CantMergeView(mesg=f'Cannot merge a view {self.iden} than has not been forked')
 
+        if self.trigqueue.size and not force:
+            raise s_exc.CantMergeView(mesg=f'There are still {self.trigqueue.size} triggers waiting to complete.', canforce=True)
```

Comment:
log the event that we're dropping trigger queue items on the floor

Reply:
If they specified force, that constitutes totally normal operations. I'd like to avoid logspam.